src/client/components/pages/Courses/MeetingTimesList.tsx

```diff
@@ -317,6 +322,22 @@ export const MeetingTimesList
                           Room:
                           {currentEditMeeting.room
                         && currentEditMeeting.room.name}
+                          {currentEditMeeting.room && (
+                            <BorderlessButton
+                              alt={`Remove Room ${index + 1} on ${meetingTimeString}${meetingRoomString}`}
+                              id={`remove-room-${meeting.id}`}
```

Comment:
Is the `id` prop being used here?

Reply:
It's not being used but I thought it would be good to have a unique identifier for the button. Is this not necessary?

Reply:
I don't believe it's necessary, unless you need to programatically refer to the button somehow from somewhere else....

Reply:
Ok thanks - originally I had been using `document.getElementById` in my tests, but this was replaced by `findByLabelText.`